cf-agent/verify_files_utils.c

```diff
@@ -2257,7 +2257,6 @@ PromiseResult VerifyFileAttributes(EvalContext *ctx, const char *file, struct st
 
     if (attr.acl.acl_entries)
     {
-        oleprint("Will call VerifyACL!");
```

Comment:
Try to remove completely your commits with your personal debugging helpers. `git rebase -i` and its `squash` subcommand are your friends there.